backend.go

```diff
@@ -16,9 +16,7 @@ const (
 )
 
 func Factory(ctx context.Context, conf *logical.BackendConfig) (logical.Backend, error) {
-	b := &backend{
-		logger: hclog.Default(),
```

Comment:
I stopped using this in favor of Vault's built in one so it would pick up user settings like the log level.